src/components/stepper/interfaces/step-config.ts

```
@@ -0,0 +1,30 @@
+
+  /** For the indicator, could be string for Waze font icon class, or a component to render as the icon */
+  icon?: string | IconDetail | ReactElement;
+
+  /** The main content of the step */
+  content: ReactNode | ComponentType<StepContentProps>;
+
+  /** Called when the step becomes active */
+  onEnter?: () => void;
+
+  /** Called before leaving. Return false to prevent. */
+  onLeave?: () => void | boolean | Promise<void | boolean>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)